Source/FLHook/HkCbIServerImpl.cpp

```diff
@@ -1017,50 +993,8 @@ namespace HkIServerImpl
 				return;
 			}
 
-			CALL_PLUGINS_V(PLUGIN_HkIServerImpl_Login, __stdcall, (struct SLoginInfo const &li, unsigned int iClientID), (li, iClientID));
-
-
-			// check for ip ban
-			wstring wscIP;
-			HkGetPlayerIP(iClientID, wscIP);
-
-			foreach(set_lstBans, wstring, itb)
-			{
-				if (Wildcard::wildcardfit(wstos(*itb).c_str(), wstos(wscIP).c_str()))
-				{
-					HkAddKickLog(iClientID, L"IP ban(%s matches %s)", wscIP.c_str(), (*itb).c_str());
-					if (set_bBanAccountOnMatch)
-						HkBan(ARG_CLIENTID(iClientID), true);
-					HkKick(ARG_CLIENTID(iClientID));
-				}
-			}
-
-			// count players
-			struct PlayerData *pPD = 0;
-			uint iPlayers = 0;
-			while (pPD = Players.traverse_active(pPD))
-				iPlayers++;
-
-			if (iPlayers > (Players.GetMaxPlayerCount() - set_iReservedSlots))
-			{ // check if player has a reserved slot
```

Comment:
if we're getting rid of the reserved slots functionality altogether, HkSetReservedSlot/CmdSetReservedSlot/HkGetReservedSlot/CmdGetReservedSlot must go too